database/src/main/java/database/service/SqlDatabaseService.java

```diff
@@ -0,0 +1,443 @@
+package database.service;
+
+import database.entity.BaseEntity;
+import database.exception.CreationDatabaseException;
+import database.exception.DatabaseDoesNotExistException;
+import database.exception.DatabaseOperationException;
+import database.exception.DeletionDatabaseException;
+import database.exception.FailedAccessFieldValueException;
+import database.exception.IdDoesNotExistException;
+import database.exception.IdProvidedManuallyException;
+import database.exception.InvalidParameterValueException;
+import database.helper.Settings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.Field;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class SqlDatabaseService implements DatabaseService {
+    private final Settings settings = new Settings();
+    private final String dbUserName;
+    private final String dbPassword;
+    private final String databaseName;
+    private final String databaseURL;
+    private static final Logger LOG = LoggerFactory.getLogger(SqlDatabaseService.class);
+    static final String UNABLE_CREATE_TABLE = "Unable to create table. Please check if it already exists";
```

Comment:
I see you have some common constants for both `Json` and `Sql` services, please create class `ServiceConstants` and put common constants to this class. Create empty private constructor in this class (it's a good practise for classes with only constants, since we don't want to create any instances of it).